ui/lib/CreateStream.tsx

```diff
@@ -135,7 +135,7 @@ const CreateStream = (props: CreateStreamProps) => {
     (tokenData?.value || BigInt(0)) / BigInt(props.amountPerSecond),
   );
 
-  // TODO: Increase stability of deployments beyond a week
+  // TODO: Increase stability of simulations beyond a week
   const maxTimeDays: number = Math.min(Math.floor(maxTime / SECS_PER_DAY), 7); // Up to a week
```

Comment:
hmm, why is there a limit here? the limit is driven by chaosnet use case

Reply:
The comment was already there, I just changed the wording. I'll remove the comment then 👍